packages/editor/src/components/block-actions/index.js

```diff
@@ -45,22 +44,18 @@ export default compose( [
 		const rootClientId = getBlockRootClientId( props.clientIds[ 0 ] );
 
 		return {
-			firstSelectedIndex: getBlockIndex( first( castArray( props.clientIds ) ), rootClientId ),
-			lastSelectedIndex: getBlockIndex( last( castArray( props.clientIds ) ), rootClientId ),
 			isLocked: !! getTemplateLock( rootClientId ),
 			blocks,
 			canDuplicate,
 			rootClientId,
 			extraProps: props,
 		};
 	} ),
-	withDispatch( ( dispatch, props ) => {
+	withDispatch( ( dispatch, props, { select } ) => {
```

Comment:
@aduth - I don't see it on the list in #12890.

Anyways, I'm happy seeing that `select` is helping us to improve performance. We still didn't discover any downsides which is encouraging to do more refactoring like this.

Reply:
>@aduth - I don't see it on the list in #12890.

I think it was missed because I was explicitly looking for `withDispatch( ( dispatch, ownProps )` or `withDispatch( ( dispatch, {`. We apparently have more variation 😬

Reply:
> I think it was missed because I was explicitly looking for `withDispatch( ( dispatch, ownProps )` or `withDispatch( ( dispatch, {`. We apparently have more variation 😬

This is the only occurrence of the second argument being named as "props" for `withDispatch`, and best I could tell (by a regular expression search of `withDispatch\( \( dispatch, [^{o]`) there are no other variations.